### PR TITLE
Updated changelog with v0.13 content

### PR DESCRIPTION
**Pull request by g-gaston**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/changelog.md [hunk 1]
```diff
@@ -6,7 +6,17 @@ weight: 35
 
 ## Unreleased
 
+### Added
+- Workload clusters full lifecycle API support ([#1090](https://github.com/aws/eks-anywhere/issues/1090))
```
Comment (maxdrib) [inline docs/content/en/docs/reference/changelog.md:10]: maybe worth listing the providers that support this? I.e. not cloudstack :(

**Comment by ndeksa** [comment]
@g-gaston following also need to go in the changelog' Added section - 
 Support for Nutanix Provider, Support for private registry authentication, Workload clusters support for bare metal, VM Tagging support for vSphere VM's created in the cluster,  Static IPs support for Bottlerocket OS in bare metal deployment

File: docs/content/en/docs/reference/changelog.md [hunk 2]
```diff
@@ -6,7 +6,18 @@ weight: 35
 
 ## Unreleased
 
+### Added
```
Comment (maxdrib) [inline docs/content/en/docs/reference/changelog.md:9]: we also added configurability for MHC timeouts with an env var input

File: docs/content/en/docs/reference/changelog.md [hunk 3]
```diff
@@ -6,7 +6,18 @@ weight: 35
...
+- Workload clusters full lifecycle API support for vSphere and Docker ([#1090](https://github.com/aws/eks-anywhere/issues/1090))
+- Single node cluster support for Bare Metal provider
+- Cilium updated to version `v1.11.10`
+- CLI high verbosity log output is automatically included in the support bundle after a CLI `cluster` command error ([#1703](https://github.com/aws/eks-anywhere/issues/1703) implemented by [#4289](https://github.com/aws/eks-anywhere/issues/1703))
+- Ability to configure rolling upgrade for Bare Metal via `maxSurge` and `maxUnavailable` parameters
+
```
Comment (ivyostosh) [inline docs/content/en/docs/reference/changelog.md:15]:
```suggestion
- New curated packages
  - Prometheus `v2.39.1`
- New versions of curated packages
  - ADOT `v0.23.0`
  - Emissary `v3.3.0`

```

File: docs/content/en/docs/reference/changelog.md (hunk 2 above)
Comment (g-gaston) [reply docs/content/en/docs/reference/changelog.md:9]: Do you have an issue link or PR?

File: docs/content/en/docs/reference/changelog.md (hunk 3 above)
Comment (TerryHowe) [reply docs/content/en/docs/reference/changelog.md:26]: This is good, but might be nice to have `- ADOT v0.23.0 upgraded from v0.21.1`

**Comment by jonahjon** [comment]
Adding mutli-region support for all supported packages. I'll be adding the full region list to the docs.

```
      'us-east-2',
      'us-east-1',
      'us-west-1',
      'us-west-2',
      'ap-northeast-3',
      'ap-northeast-2',
      'ap-southeast-1',
      'ap-southeast-2',
      'ap-northeast-1',
      'ca-central-1',
      'eu-central-1',
      'eu-west-1',
      'eu-west-2',
      'eu-west-3',
      'eu-north-1',
      'sa-east-1'
```

File: docs/content/en/docs/reference/changelog.md (hunk 3 above)
Comment (g-gaston) [reply docs/content/en/docs/reference/changelog.md:26]: Sounds good, I'll update it for Emissary to, what was the previous version?

File: docs/content/en/docs/reference/changelog.md (hunk 2 above)
Comment (maxdrib) [reply docs/content/en/docs/reference/changelog.md:10]: https://github.com/aws/eks-anywhere/pull/4123/files

File: docs/content/en/docs/reference/changelog.md [hunk 4]
```diff
@@ -6,7 +6,30 @@ weight: 35
...
+- New Nutanix Provider
+- Support for private registry authentication
+- Workload clusters support for Bare Metal
+- VM Tagging support for vSphere VM's created in the cluster
+- Static IP support for Bottlerocket OS in Bare Metal
+- Support for new curated packages:
+  - Prometheus `v2.39.1`
+- Updated curated packages' versions:
+  - ADOT `v0.23.0`
+  - Emissary `v3.3.0`
```
Comment (binyamz) [inline docs/content/en/docs/reference/changelog.md:25]:
```suggestion
  - Emissary `v3.3.0`
  - Support for packages controller to create target namespaces [#601](https://github.com/aws/eks-anywhere-packages/issues/601)
```

File: docs/content/en/docs/reference/changelog.md [hunk 5]
```diff
@@ -6,7 +6,30 @@ weight: 35
 
 ## Unreleased
 
+## [v0.13.0](https://github.com/aws/eks-anywhere/releases/tag/v0.13.0)
+### Added
+- Workload clusters full lifecycle API support for vSphere and Docker ([#1090](https://github.com/aws/eks-anywhere/issues/1090))
+- Single node cluster support for Bare Metal provider
+- Cilium updated to version `v1.11.10`
+- CLI high verbosity log output is automatically included in the support bundle after a CLI `cluster` command error ([#1703](https://github.com/aws/eks-anywhere/issues/1703) implemented by [#4289](https://github.com/aws/eks-anywhere/issues/1703))
+- Ability to configure rolling upgrade for Bare Metal via `maxSurge` and `maxUnavailable` parameters
```
Comment (maxdrib) [inline docs/content/en/docs/reference/changelog.md:15]: and cloudstack too

File: docs/content/en/docs/reference/changelog.md (hunk 2 above)
Comment (g-gaston) [reply docs/content/en/docs/reference/changelog.md:10]: Done, let me know

**Review by vivek-koppuru — APPROVED** [review]

**Pull request merged**